packages/aws-client/src/client.ts

```diff
@@ -29,22 +24,32 @@ const memoize = (fn) => {
  * Note: The returned service objects are cached, so there will only be one
  *       instance of each service object per process.
  *
- * @param {Function} Service - an AWS service object constructor function
- * @param {string} version - the API version to use
- * @returns {Function} - a function which, when called, will return an AWS service object
+ * @param {AWS.Service} Service - an AWS service object constructor function
+ * @param {string} [version] - the API version to use
+ *
+ * @returns {Function} a function which, when called, will return an instance of an AWS service
+ * object
+ *
+ * @private
```

Comment:
Why is this private if it's the main thing exported from this file?

Reply:
To prevent it from being published in our API docs. I'd hate for anyone else to use this function.